app/assets/javascripts/discourse/templates/components/user-small.hbs

```diff
@@ -1,4 +1,7 @@
 {{#link-to 'user' user.username}}
   {{avatar user imageSize="tiny"}}
   {{user.username}}
+  {{#if user.name}}
```

Comment:
Why the `#if` statement? if `user.name` is blank wouldn't it just output nothing?